### ISSUE TITLE
Replace flow types with TypeScript?

### ISSUE THREAD

**Issue by benmccann** [A11yance/aria-query#599]
I wonder if there might be any interest in replacing the flow types with TypeScript? It would be more familiar to most users and remove the need for the `@types/aria-query` package. It looks like it wouldn't be a difficult migration

**Comment by ljharb** [comment]
Yes please. I’d prefer tsdoc comments that pull types from hand-written dts files.

If there’s a way to generate flow files from those, so it’s not a breaking change for Facebook, that’d be ideal.
